.circleci/config.yml

```diff
@@ -26,8 +26,13 @@ jobs:
               tar xzf google-cloud-sdk-*.tar.gz
               google-cloud-sdk/install.sh -q
               google-cloud-sdk/bin/gcloud components install beta cloud-datastore-emulator -q
-              google-cloud-sdk/bin/gcloud config set project testing
             fi
+      - run:
+          name: "Install kubectl "
+          command: |
+            curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
I would like to verify that the downloaded file has the proper shasum as I saw honnix doing in another repo
```Dockerfile
RUN curl --silent --fail --show-error --location --output /bin/grpc_health_probe "https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.3.2/grpc_health_probe-linux-amd64" && \
--
  | chmod +x /bin/grpc_health_probe && \
  | echo 'dc5c8cd7b33ef503f49eb639af6017f33d340884  /bin/grpc_health_probe' > .grpc_checksum && \
  | sha1sum -c .grpc_checksum
```
But we won't able to fetch the latest kubectl but ping it to a version

Reply:
Does `gcloud components install kubectl` work?

Reply:
If it does, we don't need this step as it can be in previous step when installing gcloud.

Reply:
I will check if the gcloud install kubeclt works